addons/product/models/product.py

```diff
@@ -395,7 +393,6 @@ def write(self, values):
             self.invalidate_cache()
             # `_get_first_possible_variant_id` depends on variants active state
             self.clear_caches()
-        return res
```

Comment:
need to keep it

Reply:
Good catch! That explains why it was originally there twice on the cache PR when I rebased them.